packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -4,6 +4,8 @@ import chalk from 'chalk';
 import { createDeployment, createLegacyDeployment } from 'now-client';
 import wait from '../output/wait';
 import createOutput from '../output';
+import semver from 'semver';
+import pkg from '../../../package.json';
```

Comment:
There's a helper module for this:

```suggestion
import pkg from '../pkg';
```